caffe2/operators/quantized/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# ---[ CPU files only
+list(APPEND Caffe2_CPU_SRCS
+	"${CMAKE_CURRENT_SOURCE_DIR}/init_qnnpack.cc"
```

Comment:
Sorry, tabs found - could you convert that to spaces?

Reply:
fixed